packages/components/date-picker/src/use-date-picker.ts

```diff
@@ -173,6 +174,8 @@ export function useDatePicker<T extends DateValue>({
           ),
         }),
       },
+      shouldCloseOnInteractOutside: (element: Element) =>
+        ariaShouldCloseOnInteractOutside(element, domRef, state),
```

Comment:
@wingkwong check this for all implementations

```suggestion
      shouldCloseOnInteractOutside: (element: Element) =>
        chain(popoverProps.shouldCloseOnInteractOutside, ariaShouldCloseOnInteractOutside(element, domRef, state)),
```

Reply:
can't use chain since it takes boolean. revised to take popoverProps.shouldCloseOnInteractOutside first